client/styles/input.css

```diff
@@ -12,6 +12,10 @@ body {
     @apply relative w-48 border border-gray-200 p-2.5 rounded-md bg-gray-800 text-gray-200 hover:bg-gray-700 m-1;
 }
 
+.select {
```

Comment:
I guess the intention was that they probably shouldn't be the same class, but it can be, and maybe we can change it from 'button' to something else.. i mean is the dropdown technically a button? Maybe